command/task_create.go

```diff
@@ -274,9 +274,18 @@ func handleDeprecations(ui mcli.Ui, tc *config.TaskConfig) error {
 				action := generateServiceModuleInputBlockAction(tc.Services)
 				ui.Output(generateServiceFieldMsg(action))
 			case *config.ScheduleConditionConfig:
-				action := generateServicesAction(tc.Services)
+				action := generateServiceModuleInputBlockAction(tc.Services)
 				ui.Output(generateServiceFieldMsg(action))
 			}
+		} else if tc.ModuleInputs != nil {
+			for _, si := range *tc.ModuleInputs {
+				switch si.(type) {
+				case *config.ServicesModuleInputConfig:
+					action := `"list of 'services' and 'module_input "services"' block cannot both be configured. ` +
```

Comment:
Similar here, looks like there's an extra double-quote at the beginning of the string